Sources/ParseLiveQuery/Internal/ClientPrivate.swift

```diff
@@ -209,7 +209,7 @@ extension Client {
             let jsonEncoded = operation.JSONObjectRepresentation
             let jsonData = try NSJSONSerialization.dataWithJSONObject(jsonEncoded, options: NSJSONWritingOptions(rawValue: 0))
             let jsonString = String(data: jsonData, encoding: NSUTF8StringEncoding)
-
+            
```

Comment:
I just checked your current source to my current source and on line 212 there is a line....on both.